### PR TITLE
Fixing attribute that was renamed

### PR DESCRIPTION
**Pull request by bergerhoffer**
Preview: 
* https://deploy-preview-42973--osdocs.netlify.app/openshift-enterprise/latest/support/remote_health_monitoring/opting-out-of-remote-health-reporting#telemetry-consequences-of-disabling-telemetry_opting-out-remote-health-reporting
* https://deploy-preview-42973--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-6-release-notes.html
* https://deploy-preview-42973--osdocs.netlify.app/openshift-enterprise/latest/architecture/architecture-installation.html#installation-process_architecture-installation

### CODE REVIEW

File: modules/installation-overview.adoc [hunk 1]
```diff
@@ -101,7 +101,7 @@ installation program from
 ifndef::openshift-origin[]
 the appropriate
 link:https://cloud.redhat.com/openshift/install[Infrastructure Provider]
```
Comment (EricPonvelle) [inline modules/installation-overview.adoc:103]: I think cloud.redhat.com should be console here.

File: release_notes/ocp-4-6-release-notes.adoc [hunk 2]
```diff
@@ -17,7 +17,7 @@ Red Hat {product-title} (link:https://access.redhat.com/errata/RHBA-2020:4196[RH
 
 Red Hat did not publicly release {product-title} 4.6.0 as the GA version and, instead, is releasing {product-title} 4.6.1 as the GA version.
 
-{product-title} {product-version} clusters are available at https://cloud.redhat.com/openshift. The {cloud-redhat-com} application for {product-title} allows you to deploy OpenShift clusters to either on-premise or cloud environments.
+{product-title} {product-version} clusters are available at https://cloud.redhat.com/openshift. The {cluster-manager-first} application for {product-title} allows you to deploy OpenShift clusters to either on-premise or cloud environments.
```
Comment (EricPonvelle) [inline release_notes/ocp-4-6-release-notes.adoc:20]: s/cloud/console

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

**Comment by bergerhoffer** [comment]
URL fixes will go in a separate PR

**Pull request merged**